qiskit_machine_learning/kernels/fidelity_statevector_kernel.py

```diff
@@ -97,7 +97,7 @@ def __init__(
         self._auto_clear_cache = auto_clear_cache
         self._shots = shots
         self._enforce_psd = enforce_psd
-
+        self.cache_size = cache_size
```

Comment:
Should this be `self._cache` ie something we consider private and uses should not change. At present its a public instance var and part of the public API - something we document via Attributes in this case - but in this case its not something I think we want users to change other that via the constructor.

Reply:
yes it should be _cache_size, changed now